hc/local_settings.py

```diff
@@ -0,0 +1,20 @@
+import os
+
+DATABASES = {
```

Comment:
These settings are tied to your development environment.  Since we are using dj_database_url I don't think this is needed in the project